runtime/shared_common/OSCachesysv.cpp

```diff
@@ -2021,6 +2021,7 @@ SH_OSCachesysv::getJavacoreData(J9JavaVM *vm, J9SharedClassJavacoreDataDescripto
 	}
 #endif
 	descriptor->cacheDir = _cachePathName;
+	descriptor->nattach = cacheInfo.nattach;
```

Comment:
I don't expect this to compile: `cacheInfo` is not defined on Windows, and if it were, it would not have been given a value.